weblate_web/templates/snippets/vat.html

```diff
@@ -1,5 +1,5 @@
 {% load i18n %}
 <p>
-    {% blocktrans with "mailto:sales@weblate.org" as contact %}Weblate accepts payments by credit card, bank transfer or bitcoin transaction. Please <a href="{{ contact }}">get in touch</a> for more details and options.{% endblocktrans %}
-    {% trans "All prices exclude VAT, invoices for EU countries are issued as a reverse charge, invoices issued to the Czech Republic include additional VAT." %}
+    {% blocktrans with "mailto:sales@weblate.org" as contact %}Payments by credit card, bank transfer or bitcoin transaction are accepted. Please <a href="{{ contact }}">get in touch</a> for more details and possibilities.{% endblocktrans %}
+    {% trans "All prices exclude VAT, except for invoices issued to the Czech Republic. Invoices to EU countries are issued as a reverse charge." %}
```

Comment:
Hmm, this is wrong with some recent changes, EU end users will be charged VAT, so this needs more rewrite...